packages/suite/src/utils/suite/date.ts

```diff
@@ -84,14 +85,18 @@ export const calcTicksFromData = (data: { time: number }[]) => {
 };
 
 /**
- * Returns Blockbook-safe current unix timestamp (current time - 3 mins).
+ * Returns Blockbook-safe current unix timestamp (time - 3 mins).
  * Little workaround for Blockbook as it doesn't return rates data for too recent timestamps.
  *
  * @returns
  */
-export const getBlockbookSafeTime = () => {
-    const timestamp = getUnixTime(new Date());
-    return timestamp - 180; // current time - 3 mins
+export const getBlockbookSafeTime = (ts?: number) => {
```

Comment:
nit:
```suggestion
export const getBlockbookSafeTime = (timestamp?: number) => {
```
Again I would rather see here `timestamp` instead of acronym

Reply:
but what about all the bytes we could save! 😁  (you can add it to the issue)

Reply:
done